repo/meta/schema/package-schema.json

```diff
@@ -22,25 +22,32 @@
       "properties": {
         "packagingVersion": {
           "type": "string",
-          "enum": ["2.0"]
+          "enum": ["2.0"],
+          "description": "The version of the packaging format. Please do not use this version. It is only here for backwards compatibility. We recommend you use the latest version."
         },
         "name": {
-          "type": "string"
+          "type": "string",
+          "description": "The name of the package."
         },
         "version": {
-          "type": "string"
+          "type": "string",
+          "description": "The version of the package. We recommend you use semantic versions."
```

Comment:
Please give an example so users can follow a standard moving forward. There are packages which use semantic version and some with a format : serviceVersion-packagingVersion